integration-tests/pages/detailsPage.js

```diff
@@ -3,6 +3,20 @@ const relocationAndInjuries = require('./relocationAndInjuriesPage')
 
 export default () =>
   page('Use of force details', {
+    allRadiosYes: () => {
```

Comment:
might be better to just call fix `fillForm` - so we aren't leaking implementation details into the test.